packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json

```diff
@@ -0,0 +1,193 @@
+                        "label": "Heart of Ice",
+                        "value": "heart-of-ice"
+                    },
+                    {
+                        "label": "Ice Glide",
+                        "value": "ice-glide"
+                    },
+                    {
+                        "label": "Icy Claws",
+                        "value": "icy-claws"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "alwaysActive": true,
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.SpellEffectMantle.FrozenHeart.RollOptionLabel.Second",
+                "option": "frozen-heart-second",
+                "suboptions": [
+                    {
+                        "label": "Chilling Ice",
+                        "predicate": [
+                            {
+                                "not": "frozen-heart-first:chilling-ice"
+                            }
+                        ],
+                        "value": "chilling-ice"
+                    },
+                    {
+                        "label": "Heart of Ice",
+                        "predicate": [
+                            {
+                                "not": "frozen-heart-first:heart-of-ice"
+                            }
+                        ],
+                        "value": "heart-of-ice"
+                    },
+                    {
+                        "label": "Ice Glide",
+                        "predicate": [
+                            {
+                                "not": "frozen-heart-first:ice-glide"
+                            }
+                        ],
+                        "value": "ice-glide"
+                    },
+                    {
+                        "label": "Icy Claws",
+                        "predicate": [
+                            {
+                                "not": "frozen-heart-first:icy-claws"
+                            }
+                        ],
+                        "value": "icy-claws"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "Resistance",
+                "predicate": [
+                    {
+                        "or": [
+                            "frozen-heart-first:chilling-ice",
+                            "frozen-heart-second:chilling-ice"
+                        ]
+                    }
+                ],
+                "type": "cold",
+                "value": 10
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "frozen-heart-first:ice-glide",
+                            "frozen-heart-second:ice-glide"
+                        ]
+                    }
+                ],
```

Comment:
This speed bonus is conditional, so you'll need a toggleable roll option for it, which you can have disabled if the `ice-glide` roll options are not active.

Reply:
Added - check when you get a moment.